src/jvmMain/kotlin/io/beatmaps/util/review.kt

```diff
@@ -135,7 +137,11 @@ fun Application.reviewListeners() {
                                         url = review.map?.let {
                                             "${Config.siteBase()}/maps/${it.id}"
                                         },
-                                        thumbnail = DiscordEmbed.HasUrl("${Config.cdnBase("", true)}/${version.hash}.jpg"),
+                                        thumbnail = DiscordEmbed.HasUrl(
+                                            review.map?.mainVersion()?.let {
```

Comment:
Have you tested this? mainVersion() relies on `BeatmapDao.versions` being populated by `complexToBeatmap` which isn't being called and as far as I can tell `complexToReview` isn't doing anything similar

Reply:
I didn't test this, you're probably right...